pkg/serverless/proxy/transport.go

```diff
@@ -0,0 +1,76 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package proxy
+
+import (
+	"bytes"
+	"errors"
+	"net/http"
+	"net/http/httputil"
+	"strings"
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+type proxyTransport struct {
+	processor InvocationProcessor
+}
+
+func (p *proxyTransport) RoundTrip(request *http.Request) (*http.Response, error) {
+	log.Debug("[proxy] new request to %s", request.URL)
+
+	// enrich the currentInvocationDetails object
```

Comment:
Per my other comment, there are a few comments here about "enriching the currentInvocationDetails object" but I don't think you are doing that any more.